Assets/Scripts/PlayerMovement.cs

```diff
@@ -53,7 +48,7 @@ void ApplyMovementFollowCamera()
       var z = Input.GetAxis("Vertical");
 
       var offset = (transform.right * x) + (transform.up * y) + (Camera.main.transform.forward * z);
-      controller.Move(offset * speed * Time.deltaTime);
+      controller.Move(offset * (_settingsManager.MovementSpeed * Time.deltaTime));
```

Comment:
the brackets aren't needed

Reply:
actually they change the multiplication order and it was recommended by rider for efficiency xd

Reply:
well that is interesting